cpg-language-typescript/src/main/kotlin/de/fraunhofer/aisec/cpg/frontends/typescript/JavaScriptLanguage.kt

```diff
@@ -42,6 +42,24 @@ open class JavaScriptLanguage : Language<TypeScriptLanguageFrontend>(), HasShort
         TypeScriptLanguageFrontend::class
     override val conjunctiveOperators = listOf("&&", "&&=", "??", "??=")
     override val disjunctiveOperators = listOf("||", "||=")
+    override val compoundAssignmentOperators =
+        setOf(
+            "+=",
+            "-=",
+            "*=",
+            "**=",
+            "/=",
+            "%=",
+            "<<=",
+            ">>=",
+            ">>>=",
```

Comment:
>>>=? See Java

Reply:
According to ECMA-262 spec, the mozilla and w3 docs this operator exists.

Reply:
Can you maybe add a link to the spec here?